src/ReactWrapper.jsx

```diff
@@ -208,7 +208,10 @@ class ReactWrapper {
       throw new Error('ReactWrapper::unmount() can only be called on the root');
     }
     this.single('unmount', () => {
-      this.component.setState({ mount: false });
+      this.component.setState({
+        mount: false,
+        lastRender: this.component._reactInternalInstance._renderedComponent.getPublicInstance(),
```

Comment:
That makes sense. It was my first thought to put it there because the state was being modified for `mount` so it seemed like the simplest way to handle it. It could also potentially be a property of the component itself something like:
`this.component._lastRender = ...`